package.json

```diff
@@ -38,5 +38,8 @@
     "rollup-plugin-node-resolve": "^3.4.0",
     "rollup-plugin-terser": "^3.0.0",
     "semver": "^5.5.1"
+  },
+  "dependencies": {
+    "commander": "^2.19.0"
```

Comment:
Before:
```
mattdavi02m:ensure-node-env matthewdavidson$ npx gzip-size-cli dist/cli.js
npx: installed 51 in 4.241s
5.09 kB
```

After:
```
mattdavi02m:ensure-node-env matthewdavidson$ npx gzip-size-cli dist/cli.js
npx: installed 51 in 1.976s
8.76 kB
```

Reply:
Yep, I wasn't sure what to do with the built artifacts (I didn't even commit them), as I didn't know whether you preferred to generate them yourselves before publishing.

Shall I rebuild them, gzip them as you're showing here, and commit them?

Reply:
I mostly included them here for audit trail - no need for action.